src/utils.js

```diff
@@ -50,6 +50,35 @@ export function toQueryString (entry) {
     .replace(R20, '+')
 }
 
+export function toSortedQueryString (entry) {
```

Comment:
Can we move this to the test folder since the main code doesn't use it? The test utility doesn't get minified with the rest of mappersmith

Reply:
Good idea :) Plus it's a little hacky anyway...